README.developer

```diff
@@ -241,7 +241,7 @@ flake8 can be installed following the instructions here:
 
 Run it with:
   --max-line-length 80 --exclude *_pb2.py,.git,__pycache \
-  --ignore E111,E121,E127,E129 data/rdm include/ola python scripts \
+  --ignore E111,E114,E121,E127,E129,W504 data/rdm include/ola python scripts \
```

Comment:
This should match https://github.com/OpenLightingProject/ola/blob/0.10/.travis-ci.sh#L247

Would you also mind adding a note to the Travis script reminding editors to keep README.developer updated, at least until something equivalent to #691 is in.

Reply:
I added the comment.

In the build with py3 change I actually added a testcase for make check that runs python compileall to make sure all of the py at least compiles.  Would be pretty easy to do the same for flake8 since it's available via pypi.

Reply:
Sounds good although I suspect the flake8 thing should be part of the lint target rather than check, as per https://github.com/OpenLightingProject/ola/issues/691 because it's nice to encourage people to run make check, but we don't want to force them to have flake8 installed. Although for bonus points configure could handle it all seamlessly. Although I suspect it should still be a separate target, I often write rather messy code while developing, but still running make check, which I wouldn't want to fail on that, before tidying up ready for committing.

Reply:
I'll resolve this comment as I've opened #1619 to track that feature request.